src/main/java/edu/cornell/kfs/sys/service/impl/ErrorMessageUtilsServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package edu.cornell.kfs.sys.service.impl;
+
+import java.text.MessageFormat;
+
+import org.apache.commons.lang.StringUtils;
+import org.kuali.rice.core.api.config.property.ConfigurationService;
+
+import edu.cornell.kfs.sys.service.ErrorMessageUtilsService;
+
+public class ErrorMessageUtilsServiceImpl implements ErrorMessageUtilsService{
+    protected ConfigurationService configurationService;
+    
+    public String createErrorString(String errorKey, String... params) {
```

Comment:
I'm trying to understand the purpose of this. Do you anticipate sending in errorKey values that aren't actually keys but error messages themselves?